src/routes/index.svelte.md

```diff
@@ -21,7 +21,7 @@ Manjaro has a track record of pretty poor security.
 
 ### SSL Certificates
 
-[Manjaro have let their certificates expire twice!](https://redd.it/4inrut/)
+[Manjaro has let their certificates expire twice!](https://redd.it/4inrut/)
```

Comment:
have is correct, this is plural...?

Reply:
This is a typo:
- [x] Manjaro is singular
- [x] Third-person narrative is used

Reply:
manjaro is plural; im referring to the organisation of people, not the distro

Reply:
actually im dumb as fuck

Reply:
> manjaro is plural in this context; im referring to the organisation of people, not the distro

Kinda makes sense :trollface: